README.md

```diff
@@ -52,6 +52,38 @@ To run the unitests run the following command in the terminal (while the virtual
 
 `python -m unittest discover tests`
 
+## Deploy with Docker
+First things first: 
+- you need [Docker](https://docs.docker.com/install/) installed.
+- you need [Docker Hub account](https://hub.docker.com/signup)
+
+If you're on Mac or Windows, we recommend installing Docker Desktop for [Mac](https://docs.docker.com/docker-for-mac/) / [Windows](https://docs.docker.com/docker-for-windows/)
+
+### Login to DockerHub from the terminal
+After installing and creating an account, you'll also need to login to Docker from
+your terminal
+
+`docker login`
+
+> Replace $username with your Docker account username.
+
+### Build image
+To build a new Docker image from you project
+`docker build -t $username/mentorship-backend .`
+
+### Run image
+Having built the image, you probably want to run it
+`docker run -it -p 5000:5000 $username/mentorship-backend`
+
+### Access container
+By default, server runs in the container on `http://0.0.0.0:5000/`.
+
+### Share your containers with the world
```

Comment:
Great work with the details here! But we need specific instructions on how to setup and run, rather than what can we do (documentation). Readme should be focussed on "what to do" rather that "what can we do". 
So, 'Sharing your containers' section is I guess not required in project readme.

Reply:
Thanks, removed :)